src/services/hasura/hasura.service.ts

```diff
@@ -455,14 +397,6 @@ export class HasuraService {
         }
       }
     `;
-
-    console.log(query);
-
-    try {
```

Comment:
@Xitija -- why are we removing all try-catch blocks? How do we ensure to avoid unexpected error handing?